content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,195 @@
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+        numTrustedProxies: <VALUE>
+{{< /text >}}
+
+For example, if you have a cloud based Load Balancer, a reverse proxy, and an Istio gateway proxy,
+then `<VALUE>` would be 2.
+
+{{< idea >}}
+Note that all proxies in front of the Istio gateway proxy must parse HTTP traffic and append the `X-Forwarded-For`
+headers at each hop. If the number of entries in `X-Forwarded-For` header is less than the number of
+trusted hops configured, Envoy falls back to using the immediate downstream address as the trusted
+client address. Please refer to the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for)
+to understand how `X-Forwarded-For` headers and trusted client addresses are determined.
+{{< /idea >}}
+
+#### Example using X-Forwarded-For capability with httpbin
+
+1. Specify `numTrustedProxies` as 2 either through `MeshConfig` or an `proxy.istop/io/config` annotation. If you are using `MeshConfig`, run the following command to create a file named `topology.yaml` and apply it to your cluster:
+
+    {{< text bash >}}
+    $ cat << EOF > topology.yaml
+      apiVersion: install.istio.io/v1alpha1
+      kind: IstioOperator
+      spec:
+        meshConfig:
+          defaultConfig:
+            gatewayTopology:
+              numTrustedProxies: 2
+      EOF
+    $ istioctl manifest apply -f topology.yaml
+    {{< /text >}}
+
+    {{< idea >}}
+    If you previously installed an Istio ingress gateway, restart all ingress gateway pods after step 1.
+    {{</ idea >}}
+
+1. Create an `httpbin` namespace, using the following command:
+
+    {{< text bash >}}
+    $ kubectl create namespace httpbin
+    namespace/httpbin created
+    {{< /text >}}
+
+1. Set the `istio-injection` label to `enabled` for sidecar injection, using the following command:
+
+    {{< text bash >}}
+    $ kubectl label --overwrite namespace httpbin istio-injection=enabled
+    namespace/httpbin labeled
+    {{< /text >}}
+
+1. Deploy `httpbin` in the `httpbin` namespace, using the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -n httpbin -f samples/httpbin/httpbin.yaml
+    {{< /text >}}
+
+1. Deploy a gateway associated with `httpbin`, using the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -n httpbin -f samples/httpbin/httpbin-gateway.yaml
+    {{< /text >}}
+
+1. Set a local `GATEWAY_URL` environmental variable based on your Istio ingress gateway's IP address, using the following command:
+
+    {{< text bash >}}
+    $ export GATEWAY_URL=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    {{< /text >}}
+
+1. Run the following `curl` command to verify the `X-Envoy-External-Address` and `X-Forwarded-For` are set correctly, using the following command:
```

Comment:
```suggestion
1. Run the following `curl` command to verify the `X-Envoy-External-Address` and `X-Forwarded-For` are set correctly:
```

Reply:
See https://github.com/istio/istio.io/pull/7242#discussion_r424764235

I can make your suggestion if you still want it to be done.